### ISSUE TITLE
There is a significant gap in the experimental results

### ISSUE THREAD

**Issue by Luolin2333** [ASolitaryMan/HFLEA#2]
I ran your paper, but the results are significantly different from those in your work. How did you divide the test set and validation set? Are there any other possible reasons for the discrepancy?

**Comment by ASolitaryMan** [comment]
I am very sorry for my mistake. I double check the method for dataset division. I follow the leave one session out cross validation method https://github.com/ECNU-Cross-Innovation-Lab/ShiftSER. Four sessions are used as training data, and one session for testing set. I will mark errors in the readme.
